### ISSUE TITLE
multifile editor accessiblity issues

### ISSUE THREAD

**Issue by ilenia-magoni** [freeCodeCamp/freeCodeCamp#60704]
We have reports of the multifile editor being really difficult to use using a screen reader

https://forum.freecodecamp.org/t/how-to-get-css-and-html-in-the-code-editor-on-free-code-campdesign-a-business-card-design-a-business-card/749609

**Comment by huyenltnguyen** [comment]
If I understand correctly, the issue here is, if the challenge has multiple editors the user can always focus on the first and can't move to the second (or third).

- I tested on [Todo App - step 37](https://www.freecodecamp.org/learn/full-stack-developer/workshop-todo-app/step-37) with screen reader off, and I wasn't able to focus on the second and third editors by pressing `Esc` (to move focus out of the current editor) and `Tab` as the focus would move back up to the action button row.

    <details>
      <summary>Screen recording</summary>

    https://github.com/user-attachments/assets/03d7f27c-4d80-458a-b5b8-78b83f0232cd
    </details>

- I tested on [Todo App - step 37](https://www.freecodecamp.org/learn/full-stack-developer/workshop-todo-app/step-37) with screen reader (VoiceOver) on, and I was able to move focus to the second and third editors, but the navigation requires moving through the page's element one-by-one (by the DOM order).

    <details>
      <summary>Screen recording</summary>
 
   https://github.com/user-attachments/assets/5fc37b92-f714-4fa8-9bb2-5c2937c41e38

    </details>

---

I think the issue should be fixed for both screen reader and non-screen-reader users.

It looks like the `Ctrl`/`Cmd` + `M` shortcut is supposed to handle the Tab behavior here, but I'm not able to trigger it. Pressing `Cmd` + `M` would minimize the browser window for me. The condition also seems to be incorrect — looks like it should be the other way around, but I haven't played around with it locally yet.

https://github.com/freeCodeCamp/freeCodeCamp/blob/f4dc57bc505f4876384e8d342c43f41e003358d0/client/src/templates/Challenges/classic/editor.tsx#L503-L514

**Comment by huyenltnguyen** [comment]
Well actually, I was wrong. I was confused by the `WinCtrl` key in the code. It's actually the `Control` button on Mac. 

The tab focus mode is working as expected for me. On MacOS, this behavior needs to be toggled using the `Control` + `Shift` + `M`, and I'm able to move to the adjacent editors with the Tab key. Though we can only tell if the toggle has been made when we use a screen reader.

<details>
  <summary>Screen recording</summary>

https://github.com/user-attachments/assets/d4915278-0e31-4cc4-8c3d-b0972af50c88

</details>

The monaco editor has a built-in guide that is open when campers presses `Alt` + `F1` (or `Option` + `F1` on MacOS), and the guide does have the tab focus keyboard shortcut:

<details>

  <summary>Screenshot</summary>

<img width="351" alt="Image" src="https://github.com/user-attachments/assets/b493be8f-ab07-40bc-9eff-b9c392e9913c" />

</details>

**Comment by huyenltnguyen** [comment]
Closing as the issue has been resolved.

To summarize, another screen reader user helped confirm that they were able to navigate through the editors without issues, and they tested on Narrator, NVDA, and JAWS. This appeared to be an issue in the OP's screen reader setting and they have resolved that on their end.

I'll create a separate issue for potential new keyboard shortcuts for switching editors - for non screen reader users. We're able to tab to the adjacent editors, but the keyboard combination to enable that behavior is a little awkward to use.
